pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/RuntimeUtils.java

```diff
@@ -139,6 +141,14 @@ public static List<String> getGoInstanceCmd(InstanceConfig instanceConfig,
             goInstanceConfig.setClusterName(instanceConfig.getClusterName());
         }
 
+        if (null != stateStorageServiceUrl) {
+            goInstanceConfig.setStateStorageServiceUrl(stateStorageServiceUrl);
+        }
+
+        if (instanceConfig.isExposePulsarAdminClientEnabled() && StringUtils.isNotBlank(pulsarWebServiceUrl)) {
+            goInstanceConfig.setPulsarWebServiceUrl(pulsarWebServiceUrl);
```

Comment:
When it was added to functions, the admin endpoint was not exposed to the go/python languages because they do not have an admin client. That is re-iterated here:

https://github.com/apache/pulsar/blob/f04252b84a8b594ea371645e367c491aa8e2dd80/pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/RuntimeUtils.java#L407-L410

That being said, any objection to passing the admin endpoint would seem inconsistent with the fact that we are already passing it to the Java runtime. As such, I think this is a reasonable expansion.

Reply:
Oh, right, that makes it an even more obvious addition. Thanks for pointing that out.